docs/pages/database-access/enroll-aws-databases/rds.mdx

```diff
@@ -136,7 +140,50 @@ Next, get your environment ready to run the Teleport Database Service:
 <Tabs>
 <TabItem label="Linux Host">
 
-(!docs/pages/includes/database-access/aws-create-db-configuration.mdx type="RDS" matcher="rds" protocol="&lt;Var name&equals;&quot;mysql or postgres&quot;&sol;&gt;" example="my-aurora-cluster.abcdefghijklm.us-west-1.rds.amazonaws.com:5432" !)
+Install Teleport on the host where you will run the Teleport Database Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Generate a configuration for the Teleport Database Service:
+
+Generate a configuration file for the Teleport Database Service. Run one of the
+following commands, depending on whether you want to proxy an AWS Aurora or
+Amazon RDS instance. 
+
+Assign <Var name="example.teleport.sh:443" /> to the host **and port** of your
+Teleport Proxy Service or Enterprise Cloud site. Assign <Var name="us-west-1" />
+to the region associated with the RDS database.
+
+Run the following command to proxy an AWS Aurora cluster, assigning <Var
+name="RDS_URI" /> to your cluster's domain name and port and <Var
+name="protocol" /> to either `postgres` or `mysql`, depending on the database
+you want to proxy:
+
+```code
+$ sudo teleport db configure create \
+   -o file \
+   --proxy=<Var name="example.teleport.sh:443" />  \
```

Comment:
Sorry if i missed this earlier, `teleport db configure create` requires a `--name` for the database name to be registered like `postgres-rds`.

Nit: One optional parameter we could also add is `--labels env=dev`. It's a good practice to have some labels for the database. However, since we currently assigning the default `access` role which matches wildcard labels, `--labels` is not necessary. If we do add the `--labels env=dev`, we will have to add like `env=dev` in the `Labels` column of `tsh db ls`. So probably easier without it.